nexus/configure.sh

```diff
@@ -228,7 +238,11 @@ echo_info "Install Blob Stores"
 runJsonScript "createBlobStores"
 
 echo_info "Configure proxy if applicable"
-runJsonScript "createProxySettings"
+sed "s|@http_proxy@|${HTTP_PROXY}|g" json/createProxySettings.json > json/createProxySettingsWithProxy.json
+sed -i "s|@https_proxy@|${HTTPS_PROXY}|g" json/createProxySettingsWithProxy.json
+sed -i "s|@no_proxy@|${NO_PROXY}|g" json/createProxySettingsWithProxy.json
+cat json/createProxySettingsWithProxy.json
+runJsonScript "createProxySettings" "-d @json/createProxySettingsWithProxy.json"
 
```

Comment:
Could you delete the file as below? `rm json/developer-user-with-password.json`?